cmd/minikube/cmd/start.go

```diff
@@ -199,6 +212,18 @@ func runStart(cmd *cobra.Command, args []string) {
 	} else {
 		fmt.Println("Kubectl is now configured to use the cluster.")
 	}
+
+	if config.VMDriver == "none" {
```

Comment:
Let's just write the kubeconfig to the root user's home and print out a message on how to use it from another user.